gazebo/rendering/DepthCamera.hh

```diff
@@ -92,6 +92,14 @@ namespace gazebo
       /// \return The z-buffer as a float array
       public: virtual const float *DepthData() const;
 
+      /// \brief Get whether reflectance has been generated.
+      /// \return True if reflectance is set to generate.
+      public: bool GetOutputReflectance() const;
+
+      /// \brief All things needed to get reflectance data
+      /// \return The reflectance buffer as a float array
+      public: virtual const float *ReflectanceData() const;
```

Comment:
marking this `virtual` breaks ABI

* [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=gazebo-abichecker-any_to_any-ubuntu_auto-amd64&build=804)](https://build.osrfoundation.org/job/gazebo-abichecker-any_to_any-ubuntu_auto-amd64/804/) https://build.osrfoundation.org/job/gazebo-abichecker-any_to_any-ubuntu_auto-amd64/804/API_5fABI_20report/

can the `virtual` specifier be removed or is it needed?

Reply:
I added the virtual to match the method above, `public: virtual const float *DepthData() const;`, but it is not necessary. I removed the virtual modifier and pushed an update.